README.md

```diff
@@ -0,0 +1,43 @@
+## Microsoft Graph Python Client Library
+
+The Microsoft Graph Python client library is a lightweight wrapper around
+the Microsoft Graph API.
+
+## Getting Started
+
```

Comment:
Which minimum version of Python should we have installed?

Reply:
Tracked in #34